ruth/data/map.py

```diff
@@ -56,10 +57,18 @@ def __init__(self, border, data_dir="./data", with_speeds=True):
 
         if fresh_data:
             self._store()
-        self.hdf5_file_path = str((Path(data_dir) / "map.hdf5").absolute())
-        self.osm_to_hdf_map_ids = self.to_hdf5(self.hdf5_file_path)
+
+        self.save_hdf()
+
+    def save_hdf(self):
```

Comment:
It would be nice if this function returned the path where the map was stored, so that users of the function don't have to peek inside this class to figure out which attribute to access to find the correct path.